galter_subjects_utils/cli.py

```diff
@@ -104,10 +105,10 @@ def update_subjects(**parameters):
     """Update subjects in running instance according to deltas file."""
     deltas = [d for d in read_csv(parameters["deltas_file"])]
     logger = SubjectDeltaLogger(filepath=parameters["output_file"])
-    updater = SubjectDeltaUpdater(deltas, logger)
     keep_trace = KeepTrace(
         field=parameters.get("keep_trace_field") or None,
         template=parameters.get("keep_trace_template") or None
     )
-    updater.update(keep_trace)
+    updater = SubjectDeltaUpdater(deltas, logger, keep_trace)
+    updater.update()
```

Comment:
Oups, thought this was included in the previous one.

Reply:
Yeah me too. I must have been rebasing and force-pushing in-between :disappointed: